src/Core/src/Platform/iOS/LabelExtensions.cs

```diff
@@ -88,9 +88,16 @@ internal static void UpdateTextHtml(this UILabel platformLabel, ILabel label)
 			};
 
 			NSError nsError = new();
-#pragma warning disable CS8601
-			platformLabel.AttributedText = new NSAttributedString(text, attr, ref nsError);
-#pragma warning restore CS8601
+			CoreFoundation.DispatchQueue.MainQueue.DispatchAsync(() =>
+			{
+				platformLabel.AttributedText = new NSAttributedString(text, attr, ref nsError);
+				if (label.Handler is ILabelHandler labelHandler)
+				{
+					labelHandler.UpdateValue(nameof(ILabel.TextColor));
+					labelHandler.UpdateValue(nameof(ILabel.Font));
+				}
```

Comment:
Not sure if using handlers here is expected... Not sure if that is bad or good, but in my mind all the handler things happen in the handler or close to it. This is a few methods away now so may be surprising.